doc/build/macos.md

```diff
@@ -1,7 +1,12 @@
 ## macOS
 
 You need to have the current Xcode command line utilities installed: run `xcode-select --install` in the terminal.
+
 You will need to rerun this terminal command after each macOS update, otherwise you may run into errors involving missing libraries or headers.
+
 You will also need a 64-bit gfortran to compile Julia dependencies. The gfortran-4.7 (and newer) compilers in Homebrew work for building Julia.
+```bash 
+brew cask install gfortran
```

Comment:
You should instead just `brew install gcc`, rather than installing a `gfortran` cask.